### DISCUSSION TITLE
Node Problem Detector guidelines?

### DISCUSSION THREAD

**Discussion by turowicz** [k3s-io/k3s#10266] [Q&A]
Is there any kind of guideline on how to set up the node-problem-detector so it detects issues with the embedded containerd and other k3s-specific things?

**Comment by turowicz** [comment]
@brandond are you guys using node problem detector?

**Reply by brandond** [reply]
No, we don't have any experience to share in this space. You might consider asking on the NPD project instead?

**Reply by turowicz** [reply]
They told me to come here as the embedded containerd is not a standard thing in their opinion...

**Reply by turowicz** [reply]
I see you marked your reply as teh answer but its not really adressing the issue that I've commented here with after you posted
